### PR TITLE
Bump e2e-prow to v20170807-751e7e3a, add bump script

### PR DESCRIPTION
**Pull request by zmerlynn**
Adds a script to bump kubekins-e2e when you make `kubetest` changes. Includes automated test-regen step and manual `config.yaml` change.

Canary test here (waiting for green before merge):
  https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-prow-canary/17150/

### CODE REVIEW

File: experiment/bump_e2e_image.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  --yaml-config-path=experiment/test_config.yaml \
+  --json-config-path=jobs/config.json \
+  --prow-config-path=prow/config.yaml
+bazel run //jobs:config_sort
+git commit -am "Re-gen tests with ${TAG}"
+
+# Scan for kubekins-e2e-prow:v.* as a rudimentary way to avoid
+# replacing :latest.
+sed -i "s/\/kubekins-e2e-prow:v.*$/\/kubekins-e2e-prow:${TAG}/" "${TREE}/prow/config.yaml"
+git commit -am "Bump e2e-prow to ${TAG}"
```
Comment (krzyzacy) [inline experiment/bump_e2e_image.sh]: the last two commits might can be combined.

Comment (zmerlynn) [inline experiment/bump_e2e_image.sh]: I can merge them, sure. PArt of that was to differentiate between the test re-gen and the manual bump, but if you think there's no need, it saves a commit.

Comment (zmerlynn) [inline experiment/bump_e2e_image.sh]: PTAL

Comment (krzyzacy) [inline experiment/bump_e2e_image.sh]: not have too strong preference :-) either way

**Comment by krzyzacy** [comment]
ah forgot to stamp :-)

/lgtm

**Comment by zmerlynn** [comment]
Thanks!

**Comment by zmerlynn** [comment]
Ack, I think this is about to break anything using the new GKE deployment, reverting temporarily.

**Pull request merged**